linux/keyman-config/Makefile

```diff
@@ -1,17 +1,33 @@
 #!/usr/bin/make
 
-VERSION=10.99.1
+VERSION=`cat ../VERSION`
 
 default:
 	python3 setup.py build
 
-install:
-	python3 setup.py install
+install: man
```

Comment:
I think I prefer the user to run `sudo` rather than that being built into the script/Makefile. That makes it explicit to the user and they can be sure that it is `sudo` asking for their password and not something nefarious in the script ;)

Reply:
the problem with that is that, especially at the linux/Makefile level is that you end up with generated files owned by root which can be a pain to clean up. An install step to system will always need sudo so I thought that was ok. But I don't mind doing it either way.

I could do it differently in keyman-config from the others but that feels like it would get confusing

Reply:
Okay. Ideally the steps for installation are completely separate from everything else. So only a few small steps should be needed to copy user-owned files into system-owned folders for installation. I haven't analysed the makefiles in enough detail to be sure that is what you are doing. But the pattern of `make && sudo make install` is one that I would like to see. Is that possible with the current model?

Reply:
OK, I've refactored the build.sh script to separate configure, build and install steps
now to build and install everything:

*  `make devreconf` or `make reconf` to autoreconf and set the version

then:
* `make configure` (if needed)
* `make`
* `sudo make install`
    
or:
* `make fullbuild`
* `sudo make install`

Reply:
Perfect :)